GPL/EventProbe/BPFEventsTests.cpp

```diff
@@ -1,20 +1,10 @@
-// SPDX-License-Identifier: GPL-2.0-only
+// SPDX-License-Identifier: GPL-2.0-only OR BSD-2-Clause
 
 /*
  * Copyright (C) 2021 Elasticsearch BV
  *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License as published by the Free
- * Software Foundation; version 2.
- *
- * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
- * more details.
- *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc., 51
- * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ * This software is dual-licensed under the BSD 2-Clause and GPL v2 licenses.
```

Comment:
should this be

```suggestion
 * This software is dual-licensed under the BSD 2-Clause or GPL v2.
```
since we are using `OR` in the identifier?

Reply:
IMO and works better here. 

`OR` is meant to be a boolean/math thing, while the long-form license notice is a bit less strict as it's plain English and not something meant to be understood by a machine. "and" sounds the most natural here and can be taken to mean that you can choose license1 _or_ license2.

If there's any ambiguity at all in this sentence, it's resolved by the next line which explicitly says "you may choose either one", making clear what is meant.